src/main/java/com/epam/esm/repository/TagRepository.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.esm.repository;
+
+import com.epam.esm.entities.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.util.List;
+
+@org.springframework.stereotype.Repository
+public class TagRepository implements Repository<Tag> {
+
+    public static final String SELECT_BY_ID = "SELECT * FROM tag WHERE id=?";
+    public static final String GET_ALL = "SELECT * FROM tag";
+    private JdbcTemplate jdbcTemplate;
+    private RowMapper<Tag> mapper;
+
+    @Autowired
+    public void setMapper(RowMapper<Tag> mapper) {
+        this.mapper = mapper;
+    }
```

Comment:
Please, inject the beans through the constructor. It`s more safer.
https://spring.io/blog/2007/07/11/setter-injection-versus-constructor-injection-and-the-use-of-required/